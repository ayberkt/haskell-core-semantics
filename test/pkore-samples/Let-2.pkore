let(nonRec(tmVar([type omitted], name(foo)),
           lam(tmVar([type omitted], name(foo)), var(tmVar([type omitted], name(foo))))),
    var(tmVar([type omitted], name(foo))))
