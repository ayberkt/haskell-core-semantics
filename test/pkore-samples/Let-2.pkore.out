<generatedTop> <k> val ( 2 ) </k> <store> 1 |-> indirection ( 2 ) 2 |-> closure ( name ( foo ) , var ( tmVar ( [type omitted] , name ( foo ) ) ) , .Map ) </store> <env> .Map </env> <exit-code> 0 </exit-code> </generatedTop>
